piff/config.py

```diff
@@ -49,7 +49,9 @@ def setup_logger(verbose=1, log_file=None):
         handle = logging.StreamHandler()
     else:
         handle = logging.FileHandler(log_file)
-    formatter = logging.Formatter('%(message)s')  # Simple text output
+    formatter = logging.Formatter(
+        fmt='%(asctime)s %(levelname)-8s %(message)s',
+    )
```

Comment:
I'd rather not have this be the default.  Please remove this.  People can always supply their own logger if they want this kind of verbose logging.  

Or as a possible compromise if people really want this, have the time stamped version only when logging to a file, and the simple one when logging to the screen.

Reply:
I think this was originally my commit.  I found the timestamps helpful when working with Ares on finding slow spots in the code.  I wasn't aware that supplying my own logger was an option though; how does that work?

Reply:
```
piff.piffify(config, logger)
```
So, not from the command line version of `piffify`.  But only one short step up from there.

Reply:
ok, done